src/views/payment-sheet-views/paypal-view.js

```diff
@@ -14,24 +16,56 @@ PayPalView.prototype = Object.create(BaseView.prototype);
 PayPalView.prototype.constructor = PayPalView;
 PayPalView.ID = PayPalView.prototype.ID = paymentOptionIDs.paypal;
 
+PayPalView.prototype.setLogLevel = function (paypal) {
+  var level = this.model.merchantConfiguration.paypal.logLevel;
+
+  paypal.setup({
```

Comment:
We discovered that `paypal.setup` cannot be spied on by sinon, so we wrapped it in our own function so we could test the behavior. I don't really like this, but it's the best we could come up with.